p/scripts/main.js

```diff
@@ -1080,6 +1091,9 @@ function init_shortcuts() {
 			ev.preventDefault();
 			return;
 		}
+		if (ev.shiftKey && ev.key === "?") {
+			init_shortcuts_helper();
```

Comment:
If you'll permit me one final nitpick, I'd probably just insert it here, but if not the function name is a bit deceptive now. ^_^

Reply:
See https://github.com/FreshRSS/FreshRSS/pull/6981/commits/015db724e71020c581a1f39a7ada84b23f2ec0b2